packages/core/src/odata/v4/entity-deserializer.ts

```diff
@@ -12,11 +12,12 @@ import { getLinkedCollectionResult } from './request-builder/response-data-acces
  * Entity deserializer instance for v4 entities.
  * See [[EntityDeserializerType]] for the provided methods.
  */
-const deserializer: EntityDeserializer = entityDeserializer(
+export const entityDeserializerV4: EntityDeserializer = entityDeserializer(
```

Comment:
When I look at the code it seems like you use only the `deserializeEntity` method. So is this export needed?

Reply:
Also discussed on the other PR.